### ISSUE TITLE
Import SPTAG Failed

### ISSUE THREAD

**Issue by JingyuanHe1222** [microsoft/SPTAG#396]
**Describe the bug**
After building SPTAG and passing the tests in SPTAGTest.exe, I tried to run the Tutorial.ipynb codes in juptyer notebook but it fails when:

![image](https://github.com/microsoft/SPTAG/assets/74093691/7d0649da-acde-4c59-b6f6-4069907c3c79)

I checked #75 and #55 but as the above image shows, I should have the right sys path and the right python version attached. I also check CmakeCache.txt and the python path is python 3.9. 

But I realized there might be some files missing in my \build\Release folder comparing to the error message to the above two posts. 
And upon compiling ALL_BUILD I did get failure from not finding SPTAGLib.lib despite that SPTAGTest.exe passed. 
I attached the file I have below:
![image](https://github.com/microsoft/SPTAG/assets/74093691/af531834-2c11-4636-ac72-85e6a06b7c09)

**To Reproduce**
Basically following the test of installing SPTAG:
1. install swig, boost, cmake and added to environment variables
2. clone SPTAG, do cmake as instructed, open SPTAGLib.sln in visual studio and compile ALL_BUILD
3. add to environment variables
4. run Tutorial.ipynb

**Desktop (please complete the following information):**
[windows 11 home]
[64-bit, x64]

Could any try to see what's wrong with building SPTAG? 
Thank you so much!!
